### PR TITLE
test(animations): Add bundling symbol test for animations package

### PR DESCRIPTION
**Pull request by jessicajaniuk**
This adds a full bundling animation symbols test to the test suite.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: Tests

## What is the current behavior?

No symbols test exists for animations

Issue Number: N/A

## What is the new behavior?
A symbols test now exists for animations

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/test/bundling/animations/index.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports: [BrowserModule, BrowserAnimationsModule],
+})
+class AnimationsExampleModule {
+  ngDoBootstrap(app: any) {
+    app.bootstrap(RootComponent);
+  }
+}
+
+(window as any).waitForApp = platformBrowser().bootstrapModuleFactory(
+    new NgModuleFactory(AnimationsExampleModule), {ngZone: 'noop'});
```
Comment (AndrewKushnir) [inline packages/core/test/bundling/animations/index.ts:45]: Let's use factory-less API here:
```suggestion
(window as any).waitForApp = platformBrowser().bootstrapModule(AnimationsExampleModule, {ngZone: 'noop'});
```

**Review by AndrewKushnir — APPROVED** [review]
Looks great (just one quick comment) 👍

Thanks @jessicajaniuk.

File: packages/core/test/bundling/animations/index.ts (hunk 1 above)
Comment (jessicajaniuk) [reply packages/core/test/bundling/animations/index.ts]: Updated! Thanks!

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit c9fcc4f6178645d35aae44297d2bb54845c000c9.

**Pull request closed without merging**